src/components/layout/header/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import Logo from "../../ui/logo/index";
+import HeaderLinks from "../../ui/links";
+import IconButton from "../../ui/iconbutton";
+import SocialHandler from "../../ui/social-handler";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. This import statement should be after the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine